api/well-known/README.md

```diff
@@ -0,0 +1,74 @@
+# Discovery APIs
+The APIs described here allow a user of the veraison to obtain and view meta information about the veraison deployment. Information about deployment can be queried according to the service e.g. (provisioning or verification).
+
+The relevant resource is created using in response to a client `GET`. The queried information is then outputted as shown examples below.
+
+## Well know API: Provisioning service
+Each well-know provisioning session is associated to its own resource, with the
+following attributes:
```

Comment:
there is not really a session here: everything is resolved within an idempotent transaction.

```suggestion
Each well-known resource has the following attributes:
```